takwimu/urls.py

```diff
@@ -30,6 +30,7 @@
         IndicatorsGeographyDetailView.as_view(),
         name='geography_detail'),
     url(r'^profiles/$', ProfileView.as_view(), name='profiles'),
+    url(r'^analysis/(?P<country_slug>[\w-]+)', AnalysisView.as_view(), name='analysis'),
```

Comment:
I don't think we should add another url pattern `/analysis/..`  for country analysis. We should stick to the current url `profiles/country_slug/#analysis_topic` and page `takwimu/profile_page.html`

Reply:
What will we want from wazimap profile page ? 
So far i see nothing being used atleast in the new designs.

cc @kilemensi

Reply:
The analysis pages are found under `/profiles/<country-slug>` and HURUmap pages are found under `/profiles/country-<country-code>-<country_slug>`  @karimkawambwa. They share `/profiles/` but they're completely different pages with completely different templates and view classes.